### PR TITLE
Created events page, a forum where users can discuss the Olympic events that interest them

### PR DESCRIPTION
**Pull request by katsun14**

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/EventsPageServlet.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles fetching all messages for the events feed. */
+@WebServlet("/events")
+public class MessageFeedServlet extends HttpServlet {
```
Comment (zschreibs) [inline src/main/java/com/google/codeu/servlets/EventsPageServlet.java]: This needs to be called EventsPageServlet, it's not building when I try to run it bc its saying this is a duplicate class

**Review by zschreibs — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/codeu/servlets/EventsPageServlet.java (hunk 1 above)
Comment (katsun14) [reply src/main/java/com/google/codeu/servlets/EventsPageServlet.java]: Sorry about that should be fixed!

File: src/main/webapp/events.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  <link rel="stylesheet" href="/css/events-page.css">
+  <script src="/js/events-loader.js"></script>
+  <script src="/js/navigation-loader.js"></script>
+</head>
+
+<body onload="loadNavBar();">
+  <nav>
+    <ul id="navigation">
+      <li><a href="/">Home</a></li>
+    </ul>
```
Comment (zschreibs) [inline src/main/webapp/events.html:21]: add
<li class="dropdown">
        <a href="javascript:void(0)" class="dropbtn">More</a>
        <ul class="dropdown-content" id="dropdown-content"></ul>
      </li>
beneath the home li element so the rest of the nav bar loads

File: src/main/webapp/events.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+<head>
+  <title>Events</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <link rel="stylesheet" href="/css/events-page.css">
+  <script src="/js/events-loader.js"></script>
+  <script src="/js/navigation-loader.js"></script>
+</head>
+
+<body onload="loadNavBar();">
```
Comment (zschreibs) [inline src/main/webapp/events.html]: add buildUI(); in onload so the page loads

**Review by zschreibs — COMMENTED** [review]
I'm still a little confused by this page - right now it shows me info like this:
ENTITY: Yo Type: PERSON Salience: 0.7145895 Metadata: Sentiment Score: 0.0 ENTITY: test Type: OTHER Salience: 0.28541052 Metadata: Sentiment Score: 0.0
which I don't understand how that's pertinent to the user, but if we don't have it then it's essentially the same thing as the public feed

**Review by zschreibs — APPROVED** [review]

**Pull request merged**